test/run

```diff
@@ -13,6 +13,16 @@ if [ "$TEST_SCENARIO" == "devel" ]; then
     export TEST_COVERAGE=yes
 fi
 
+if [ "$TEST_SCENARIO" == "podman-next" ]; then
+    export TEST_COPR="rhcontainerbot/podman-next"
+fi
+
+TEST_COPR=${TEST_COPR:-}
+if [[ "$TEST_SCENARIO" == *"packit"* ]]; then
```

Comment:
The quoting here is odd -- the important bit to quote is the `*`, not the "packit" string, no? I.e. should this be `"*packit*"`? (I'm not familiar with these bash specific conditions)

Reply:
bash is weird.. but mm maybe we can do [better](https://stackoverflow.com/a/52671757) ie.

"${TEST_SCENARIO}" =~ "packit"

Reply:
Ah, if that works, sure. I only know `| grep -q packit` or `[ "${TEST_SCENARIO%packit*}" != "$TEST_SCENARIO" ]`, but I fully admit that this is horrible to read -- the `=~` looks nice!